src/ai/ai.cpp

```diff
@@ -2452,17 +2452,18 @@ void make_peace_offers(sys::state& state) {
 			if(overall_score >= 0) { // attacker winning
 				auto total_po_cost = military::attacker_peace_cost(state, w);
 				if(w.get_primary_attacker().get_is_player_controlled() == false) { // attacker makes offer
-					if(overall_score >= 100 || (overall_score >= 50 && overall_score >= total_po_cost * 2)) {
+					if(overall_score >= 100.f || (overall_score >= 50.f && overall_score >= total_po_cost * po_value_factor)) {
```

Comment:
2 wasn't random. It is the point at which the other side starts getting hit with penalties for refusing